examples/react/basic/package.json

```diff
@@ -26,10 +26,10 @@
     "@testing-library/react": "^16.0.1",
     "@types/react": "^18.3.5",
     "@types/react-dom": "^18.3.0",
-    "@vitejs/plugin-react": "^4.3.1",
+    "@vitejs/plugin-react": "^4.3.2",
```

Comment:
Did we mean to update these dependencies here? Are they somehow related to protobuf updates?

Reply:
Ah yes sorry, I forgot to add context. When I tried to update protobuf version, pnpm was downloading types from two different vite versions, which was resulting in a type error in the example's vite config. I tried the usual steps of deleting the lockfile and removing node modules didn't work. So I upgraded to latest.

Reply:
Makes sense.